blkmaker.py

```diff
@@ -14,7 +14,7 @@
 
 from .blktemplate import _Transaction, request as _request
 
-MAX_BLOCK_VERSION = 1 << 29
+MAX_BLOCK_VERSION = 1 << 30 - 1
 
```

Comment:
# Refer to versionbits.h, validation.cpp, ComputeBlockVersion()
VERSIONBITS_TOP_BITS = 0x20000000
MAX_BLOCK_VERSION = VERSIONBITS_TOP_BITS * 2 - 1